en/docs/quick-start-guides/deploy-a-web-application-which-consumes-a-backend-service.md

```diff
@@ -1,12 +1,12 @@
-# Build Your First Cloud Native Application with Choreo
+# Deploy a Web Application which Consumes a Backend Service
```

Comment:
We might have to go with the term "that" instead of "which" here.

Reply:
yes we can do that